.circleci/config.yml

```diff
@@ -291,49 +291,20 @@ jobs:
           name: Verify installation
           command: vector --version
 
-  #
-  # Verifying the rpm artifact
-  #
-
-  verify-rpm-artifact-on-amazon-linux-1:
+  verify-docker:
```

Comment:
I added a `verify-docker` step to confirm that Docker builds and runs properly. This process is quick enough to add as a verification step.